lib/FundingProgrammes.php

```diff
@@ -55,6 +56,10 @@ public function transform(Entry $entry)
             'organisationType' => $entry->organisationType ?? null,
             'legacyPath' => $entry->legacyPath ?? null,
             'caseStudies' => $entry->relatedCaseStudies ? ContentHelpers::extractCaseStudySummaries($entry->relatedCaseStudies->all()) : [],
+            'projectStories' => array_map(function ($entry) {
+                $transformer = new ProjectStoriesTransformer($this->locale);
```

Comment:
Realised that these transformers are plain PHP classes so you can instantiate your own instance to power related entries stuff rather than us needing to create one-off `extract*Summary` methods or duplicating code.